packages/diagrams/frontend/sirius-components-diagrams/src/renderer/edge/EdgeLayout.ts

```diff
@@ -22,7 +22,7 @@ import {
   GetUpdatedConnectionHandlesParameters,
   NodeCenter,
 } from './EdgeLayout.types';
-import { isDescendantOf } from '../layout/layoutNode';
+import { isDescendantOf, isDescendantOfId } from '../layout/layoutNode';
```

Comment:
why did you need a new isDescendantOfId ?

Reply:
I didn't want to retrieve the entire node just to use its id